Editor/EditorCore/ProBuilderEditor.cs

```diff
@@ -567,7 +567,18 @@ void OnSceneGUI(SceneView sceneView)
 
         internal void HandleMouseEvent(SceneView sceneView, int controlID)
         {
-            if(m_CurrentEvent.type == EventType.MouseDown && HandleUtility.nearestControl == controlID)
+            if ((Event.current.modifiers & EventModifiers.Alt) == EventModifiers.Alt)
+                return;
+
+#if UNITY_2023_1_OR_NEWER
+            bool isLeftMouseClick = m_CurrentEvent.type == EventType.MouseDown && m_CurrentEvent.keyCode == KeyCode.Mouse0 && HandleUtility.nearestControl == controlID;
+#elif UNITY_2022_3
+            KeyCode keyCode = m_CurrentEvent.isMouse ? m_CurrentEvent.button + KeyCode.Mouse0 : m_CurrentEvent.keyCode;
+            bool isLeftMouseClick = m_CurrentEvent.type == EventType.MouseDown && keyCode == KeyCode.Mouse0 && HandleUtility.nearestControl == controlID;
+#else
+            bool isLeftMouseClick = m_CurrentEvent.type == EventType.MouseDown && HandleUtility.nearestControl == controlID;
```

Comment:
This case isn't testing for left button, is that intentional?

Reply:
I thought I'd leave the code as it was before the shortcut changes, but I realize that it's not necessary. Fixed in [a7935ea](https://github.com/Unity-Technologies/com.unity.probuilder/pull/487/commits/a7935ea69bab0ff81f2bef19eebaeab31d355a32)